README.md

```diff
@@ -1,7 +1,35 @@
 # ![NGI-RNAfusion](https://raw.githubusercontent.com/SciLifeLab/NGI-RNAfusion/master/docs/images/NGI-RNAfusion_logo.png)
 
+## *NB* This is work in progress - use at your own risk!
+
 Attempt at setting up a pipeline for fusion detection as an addon to the previously existing NGI-RNAseq pipeline.
```

Comment:
Might be nice to model the main readme text on what we've written for the other pipelines.. Now that it's working it's good to polish a little :)